package-lock.json

```diff
@@ -13,32 +13,33 @@
         "react": "17.0.1",
```

Comment:
if you are using `npm` then you will have this file.  If you are using `yarn` then you will not need this file.  Note that this should be mutually exclusive, not mixed.  You may develop strange problems with your build in mixed modes as build complexity increases.